Calibration/HcalAlCaRecoProducers/plugins/AlCaHcalIsotrkFilter.cc

```diff
@@ -0,0 +1,194 @@
+// system include files
+#include <atomic>
+#include <memory>
+#include <cmath>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDFilter.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/Common/interface/Ref.h"
+#include "DataFormats/HcalCalibObjects/interface/HcalIsoTrkCalibVariables.h"
+
+//#define EDM_ML_DEBUG
+//
+// class declaration
+//
+
+namespace AlCaHcalIsoTrk {
```

Comment:
According to coding rule 2.7: "For namespaces use lower case, e.g. namespace edm."

https://cms-sw.github.io/cms_coding_rules.html